docs/guide/account_and_client.rst

```diff
@@ -27,9 +27,7 @@ To enable auto estimation, set the ``auto_estimate`` parameter to ``True``.
 
     It is strongly discouraged to use automatic fee estimation in production code as it may lead to an unexpectedly high fee.
 
-The returned estimated fee is multiplied by ``1.5`` for V1 and V2 transactions to mitigate fluctuations in price.
-For V3 transactions, ``max_amount`` and ``max_price_per_unit`` are scaled by ``1.1`` and ``1.5`` respectively.
-
+The returned estimated fee is multiplied by ``1.5`` to mitigate fluctuations in price.
```

Comment:
I think this is a bit misleading, as in case of V3 transactions the overall fee is actually multiplied by 1.5*1.5

```suggestion
The returned estimated fee is multiplied by ``1.5`` for V1 and V2 transactions to mitigate fluctuations in price.
For V3 transactions, ``max_amount`` and ``max_price_per_unit`` are scaled by ``1.5`` and ``1.5`` respectively.
```